### ISSUE TITLE
Alert Emails not Sent if Authentication is not Used

### ISSUE THREAD

**Issue by zms200** [bhdresh/Dejavu#13]
The user interface (`updateSettingsView.php`) states that SMTP credentials are optional, however alert emails are not sent if authentication is not used. Upon looking at `mailAlert.php` we find the following code:

```
if($hostname && $username && $password){
                //Create a new PHPMailer instance
                $mail = new PHPMailer;
                $mail->isSMTP();
                //Enable SMTP debugging
                $mail->SMTPDebug = 2;
                $mail->Host = $hostname;
                $mail->Port = 25;
                ...
```
This if statement is not followed by and else/else if statement - therefore, the process to create and send an email is only started when a username and password are provided. I confirmed that this was causing the issue by removing ` && $username && $password` from the conditional statement, and which point I started getting email alerts as expected.

**Comment by bhdresh** [comment]
Thank you for the report, will get this covered in the new version by end of this week.

Regard,s
-Bhadresh

**Comment by zms200** [comment]
Great! Looking forward to checking out the new version!

**Comment by zms200** [comment]
Is this still expected?

**Comment by bhdresh** [comment]
Hello,

Apologies for the delay. It is addressed now in v10.0

Regards,
-Bhadresh
